protobuf/bess_msg.proto

```diff
@@ -499,6 +499,17 @@ message ConfigureGateHookRequest {
 }
 
 //  -------------------------------------------------------------------------
+//  Resume hooks
+//  -------------------------------------------------------------------------
+
+// Enable/Disable a global ResumeHook.
+message ConfigureResumeHookRequest {
+  string hook_name = 1;         /// Name of the hook
+  bool enable = 5;              /// Installs the hook if True, else uninstalls
```

Comment:
Since the protobuf has not been released, we can do 1, 2, 3, ... for the member index?

Reply:
Definitely